bears/vcs/git/GitCommitBear.py

```diff
@@ -135,7 +135,7 @@ def check_shortlog(self, shortlog,
         :param shortlog_length:          The maximum length of the shortlog.
                                          The newline character at end does not
                                          count to the length.
```

Comment:
could you please enclose in double backticks "regex" and "shortlog_regex" in the commit shortlog please? :) 
if character count goes over 50 chars, you can just ommit ".py" from the message